.github/workflows/publish.yml

```diff
@@ -0,0 +1,89 @@
+name: QPP Measures Data Publish
+
+on:
+  release:
+    types: [ prereleased ]
+
+jobs:
+
+  publish-gpr:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout Codebase
+        uses: actions/checkout@v2
+
+      - name: Configure Node version and registry
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://npm.pkg.github.com/
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Publish to GitHub Package Registry
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+  publish-npm:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout Codebase
+        uses: actions/checkout@v2
+
+      - name: Configure Node version and registry
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: Install Dependencies
+        run: npm ci
+
+      - name: Publish to NPM
+        run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
+
+  # Make an announcement to other repositories of interest
+  repo-dispatch:
+    runs-on: ubuntu-latest
+    needs: publish-npm
+    strategy:
+      matrix:
+        repo: 
+          - CMSgov/qpp-scoring-engines
+          - CMSgov/beneficiary-reporting-api
+          - CMSgov/qpp-submissions-api
+    steps:
+      - name: Repository Dispatch
+        uses: peter-evans/repository-dispatch@v1.1.0
+        with:
+          token: ${{ secrets.GITHUB_USER_TOKEN }}
+          repository: ${{ matrix.repo }}
+          event-type: lib-update-event
+          client-payload: '{"ref": "${{ github.ref }}", "private_package": "@CMSGov/qpp-measures-data", "public_package":"qpp-measures-data" ,"tag_name": "${{ github.event.release.tag_name }}", "html_url": "${{github.event.release.html_url}}"}'
+  
+  # make an announcement to relevant slack channels
+  notify-slack:
+    runs-on: ubuntu-latest
+    needs: publish-npm
+    strategy:
+      matrix:
+        channel:
+          - p-qppsf-scoring
+          - p-wi
+          - p-qppsf-api-team
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
why do we need checkout here ? 
Also in every file, use the documentation style (see the `name`  filed) 
```

      - name: Checkout Codebase
        uses: actions/checkout@v2

      - name: Configure Node version and registry
        uses: actions/setup-node@v1
        with:
          node-version: 12
          
```